CircleciScripts/unlock_emulatorscreen.py

```diff
@@ -0,0 +1,14 @@
+from functions import runcommand
+import time
+int ncount = 0 
+print("unlocking emulator screen ...")
+while True:
```

Comment:
Should this fail after some time or are we ok with this running until CircleCI times out?

Reply:
I  believe we need let this fail after some time